src/api/operations/ce.fragments.graphql

```diff
@@ -21,6 +21,15 @@ fragment CeOpportunityFields on CeOpportunity {
   }
 }
 
+fragment ClientCeOpportunitySummaryFields on CeOpportunity {
+  ...CeOpportunitySummaryFields
```

Comment:
Seems like this resolves the eligibility requirements and priority scheme too, and active referral, do we need all that? In the UI it is just showing project name and type. We shouldn't resolve things we don't need since it will slow down the page load unnecessarily.

Reply:
That said I do think it makes sense to display the opportunity name in that table, maybe as the first column. Since currently there are duplicate rows with no visible differentiators indicating that they are different units.

Reply:
Whoops my bad, I must have been looking at CeOpportunityFields by accident. Thanks!